### PR TITLE
Clean debug logs

### PR DESCRIPTION
**Pull request by peterwillcn**

### CODE REVIEW

File: dpos/state/arbitrators.go [hunk 1]
```diff
@@ -1660,13 +1661,21 @@ func (a *arbitrators) resetNextArbiterByCRC(versionHeight uint32, height uint32)
...
+		}
+
 		sort.Slice(votedProducers, func(i, j int) bool {
 			if votedProducers[i].votes == votedProducers[j].votes {
 				return bytes.Compare(votedProducers[i].info.NodePublicKey,
 					votedProducers[j].NodePublicKey()) < 0
 			}
 			return votedProducers[i].Votes() > votedProducers[j].Votes()
 		})
+		if len(votedProducers) < len(a.chainParams.CRCArbiters) {
```
Comment (Houshoupei84) [inline dpos/state/arbitrators.go:1675]: duplicated with 1664

File: log.go [hunk 2]
```diff
@@ -178,8 +178,8 @@ func setupLog(c *cli.Context, s *settings.Settings) {
 		s.Config().MaxPerLogSize, s.Config().MaxLogsSize)
 	pgBar = newProgress(logger.Writer())
 
-	admrlog := wrap(logger, elalog.LevelOff)
-	cmgrlog := wrap(logger, elalog.LevelOff)
+	admrlog := wrap(logger, elalog.Level(s.Params().PrintLevel))
```
Comment (Houshoupei84) [inline log.go:181]: log level should be revert

File: pow/revertlistener.go [hunk 3]
```diff
@@ -20,7 +20,9 @@ func (pow *Service) ListenForRevert() {
 	go func() {
 		for {
 			time.Sleep(CheckRevertToPOWInterval)
-
+			//if pow.chain.BestChain.Height < 2235 {
```
Comment (Houshoupei84) [inline pow/revertlistener.go:23]: del this

**Review by Houshoupei84 — APPROVED** [review]

File: blockchain/blockvalidator.go [hunk 4]
```diff
@@ -333,9 +333,9 @@ func (b *BlockChain) CheckBlockContext(block *Block, prevNode *BlockNode) error
 	if err := DefaultLedger.Arbitrators.CheckCustomIDResultsTx(block); err != nil {
 		return err
 	}
-	if err := DefaultLedger.Arbitrators.CheckRevertToDPOSTX(block); err != nil {
-		return err
-	}
+	//if err := DefaultLedger.Arbitrators.CheckRevertToDPOSTX(block); err != nil {
```
Comment (RainFallsSilent) [inline blockchain/blockvalidator.go:336]: need to check RevertToDPOS transaction

File: dpos/manager/dposnormalhandler.go [hunk 5]
```diff
@@ -37,6 +37,7 @@ func (h *DPOSNormalHandler) ProcessAcceptVote(id peer.PID, p *payload.DPOSPropos
 }
 
 func (h *DPOSNormalHandler) TryCreateRevertToDPOSTx(BlockHeight uint32) bool {
+	//log.Info("TryCreateRevertToDPOST: %d", BlockHeight)
```
Comment (RainFallsSilent) [inline dpos/manager/dposnormalhandler.go:40]: remove it

File: dpos/manager/proposaldispatcher.go [hunk 6]
```diff
@@ -669,17 +657,16 @@ func (p *ProposalDispatcher) OnResponseInactiveArbitratorsReceived(
...
-
 	minSignCount := int(float64(p.cfg.Arbitrators.GetArbitersCount())*
 		state.MajoritySignRatioNumerator/state.MajoritySignRatioDenominator) + 1
 	if signCount >= minSignCount {
 		payload := p.RevertToDPOSTx.Payload.(*payload.RevertToDPOS)
 		p.cfg.Arbitrators.SetNeedRevertToDPOSTX(true)
-		p.cfg.Manager.AppendToTxnPool(p.currentInactiveArbitratorTx)
+		err := p.cfg.Manager.AppendToTxnPool(p.RevertToDPOSTx)
+		if err != nil {
+			log.Warnf("#### [tryEnterDPOSState] err %s", err)
```
Comment (RainFallsSilent) [inline dpos/manager/proposaldispatcher.go:667]: remoe ####

**Review by RainFallsSilent — APPROVED** [review]

**Pull request merged**